src/test/java/com/fasterxml/jackson/core/filter/BasicParserFilteringTest.java

```diff
@@ -423,7 +423,8 @@ public void testNoMatchFiltering6() throws Exception
         assertEquals(0, p.getMatchCount());
     }
 
-    public void testValueOmitsFieldName1() throws Exception
+    // https://github.com/FasterXML/jackson-core/issues/649
```

Comment:
Could you create a stand-alone test for this, under ".../failing/"? That way it's easy to locate, can be run manually, but won't be run as part of build (same as renaming here)?

Reply:
Sure thing, didn't know about that directory. Fixed in 213b8143eef161096d11667e49de6f7e02ad8bf5